scripts/capture-hw-details.sh

```
@@ -50,3 +46,50 @@ if [ "$QUIET" = false ]; then
     echo "AGAMA_VERSION=$AGAMA_VERSION"
     echo "GPU_DEVICE=$GPU_DEVICE"
 fi
+
+agama_to_driver_type() {
```

I think we do not need to detect what driver (rolling or lts) is currently installed. Instead, a specific runner should set an environment variable either to select a skip list or to select a release stream (if you still need it for some reason).

---

@AshburnLee could you please address @pbchekin comments?

---

Thanks~ I'm working on other PR's comments.

---

So from what I understand, in this PR, I just need to: 
1. remove `agama_to_driver_type()` and `xpu_is_pvc()`, 
2. Run `test_triton.sh` and collect `FAILED` cases from the log, and put those `FAILED` in a skiplist `pvc_lts/....txt`

Please confirm that I am right @pbchekin , thank you.

---

And the automation script https://github.com/intel/intel-xpu-backend-for-triton/pull/1196 is not in this scope

---

Yes, you can use test report from https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/9507147050. This is a run where all errors are ignored and full test report is attached as artifact.

```
all: passed: 17460, failed: 129, skipped: 400, xfailed: 1240, total: 19229, fixme: 12 pass rate (w/o xfailed): 97.06%
```

---

working on it.

---

done~